sdk/appcenter-data/src/main/java/com/microsoft/appcenter/data/models/PaginatedDocuments.java

```diff
@@ -7,6 +7,7 @@
 
 import android.support.annotation.NonNull;
 
+import com.microsoft.appcenter.data.Data;
```

Comment:
clean imports, this file has a warning as well for http client.

We should commit using the Android Studio dialog to enable the options to check code analysis and imports, or at least do it manually.

Reply:
I don't use Android Studio for commits. Any other way to run the code analysis locally?

Reply:
Analyze > Inspect code.
Code > Organize imports, reformat is there too.

Reply:
Tip: You can use `git reset --soft origin/develop` to have all your PR files shown as modified so that you can run code analysis only on "non committed files".

`git reset --soft origin/your-remote-branch` to go back to a the right state.